### DISCUSSION TITLE
[FEATURE] Clustering (Upvote if want)

### DISCUSSION THREAD

**Discussion by bitnom** [valkey-io/valkey#121] [General]
maybe this is already coming but adding it nonetheless

**Comment by madolson** [comment]
What is clustering? Valkey already supports clustering.

**Reply by CanRau** [reply]
Hope valkey can support clustering like KeyDB which works nicely on fly.io where instances can just discover & connect to each other without additional services like Redis requires afaik

@madolson Maybe that's what OP's referring to?

**Reply by madolson** [reply]
Redis and Valkey don't require any additional services to discover. I'm not really sure what is the difference, it might just be a fly.io thing?

**Reply by CanRau** [reply]
Hmm interesting I was referring to `High availability with Redis Sentinel` and
`Failover redis with Consul DNS` 

But I just read in their docs `Redis Sentinel provides high availability for Redis when not using Redis Cluster.` 🥹

**Reply by madolson** [reply]
There are 8 other people who upvoted this, but I have no idea what they want :/
